b/bat/stone.yaml

```
@@ -0,0 +1,27 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : bat
+version     : 0.25.0
+release     : 1
+homepage    : https://github.com/sharkdp/bat
+upstreams   :
+    - https://github.com/sharkdp/bat/archive/refs/tags/v0.25.0.tar.gz : 4433403785ebb61d1e5d4940a8196d020019ce11a6f7d4553ea1d324331d8924
```

this becomes easier to read if you add an indented dict property:

```suggestion
    - https://github.com/sharkdp/bat/archive/refs/tags/v0.25.0.tar.gz :
        hash: 4433403785ebb61d1e5d4940a8196d020019ce11a6f7d4553ea1d324331d8924
```

Just make sure the `:` character is aligned with the other `:` characters so it looks "nice".